### ISSUE TITLE
GOLE 1 PRO 【all Driver】-Driver files-Article updated on 2023-02-15-NEW

### ISSUE THREAD

**Issue by Fangxie-max** [HIGOLEPC/GOLE-1-PRO-MINI-PC#13]
https://higole-my.sharepoint.com/:u:/g/personal/support_higole_onmicrosoft_com/EaB3m9VV4atBhMzKJcAWm2kBHvVwQ5IUNhDYc7boo78IJA?e=GbkRMz

![image](https://user-images.githubusercontent.com/115788027/218914439-d4b8fa43-5f36-4118-9167-4b3c02062ee3.png)
![image](https://user-images.githubusercontent.com/115788027/218914476-738404cf-79e6-47d4-ab07-37e11d1257e9.png)

**Comment by TSBeautyAshe** [comment]
Link is dead. :'(

**Comment by kpma1985** [comment]
2 min google


https://www.golerugged.com/supports/348.html

**Comment by TSBeautyAshe** [comment]
Yes, I have found that link already, but I can't download it because I am
in the United States and Baidu does not let people in the United States use
their service. I usually purchase Higole Mini PC products shortly after
they are released but if I can't access the OS download just because I
don't live in China it doesn't feel worth it to me because if my device
ever crashes or becomes corrupted for some reason I will have no way to
reinstall the OS or the drivers for the device. Is there any possible way
that the file for the Gole1 OS and the Gole2 OS could be uploaded to a file
sharing service that allows your United States customers to access the
downloads?

**Comment by Gejao** [comment]
I used the double driver software before formatting, this software saves all the drivers and gives you the possibility to reinstall after formatting, for those who need it I uploaded it to MEGA and I will make the link available for those who need it

https://mega.nz/file/oA013IRA#t8FVUlqhpPbW4o5aFKpuUmsa04MOfIacYP49N64d2NM

The folder containing the double driver software is in the compressed file, you can use it to do the installation, a tip, install only the essentials and let Windows Update install the rest
